crates/compiler/parse/src/src64.rs

```diff
@@ -228,6 +228,7 @@ impl<'a> Src64<'a> {
                 // chunk(s) won't be a cache miss anymore because they'll already be in cache.
                 //
                 // We can do further prefetches in the actual tokenization loop.
+                #[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
```

Comment:
This basically does the same thing as before with " If we're not on x64 or aarch64, just do nothing!" (see later in this file) but without generating warnings :)